src/erlfmt_cli.erl

```diff
@@ -56,9 +61,16 @@ do(Opts, Name) ->
 do_unprotected(Opts, Name) ->
     case parse_opts(Opts, Name, [], #config{}) of
         {format, Files, Config} ->
+            case Config#config.out of
+                check -> io:format(standard_error, "Checking formatting...~n", []);
+                _ -> ok
+            end,
             case parallel(fun(File) -> format_file(File, Config) end, Files) of
-                true -> erlang:halt(4);
-                false -> ok
+                ok -> ok;
+                warn ->
+                    io:format(standard_error, "[warn] Code style issues found in the above file(s). Forgot to run erlfmt?~n", []),
+                    erlang:halt(1);
+                error -> erlang:halt(4)
```

Comment:
What happens if we get both errors and check failures? Does the output look reasonable?

Reply:
The error trumps the warning

Reply:
So it would not print out the final warn and only print out the final error.  I think this is acceptable, what do you think?

Reply:
Yeah, I think that's fine